packages/react-reconciler/src/ReactFiberDevToolsHook.js

```diff
@@ -65,12 +52,39 @@ export function injectInternals(internals: Object): boolean {
   try {
     const rendererID = hook.inject(internals);
     // We have successfully injected, so now it is safe to set up hooks.
-    onCommitFiberRoot = catchErrors(root =>
-      hook.onCommitFiberRoot(rendererID, root),
-    );
-    onCommitFiberUnmount = catchErrors(fiber =>
-      hook.onCommitFiberUnmount(rendererID, fiber),
-    );
+    onCommitFiberRoot = (root, expirationTime) => {
+      const currentTime = requestCurrentTime();
+      const priorityLevel = inferPriorityFromExpirationTime(
+        currentTime,
+        expirationTime,
```

Comment:
Should these be DEV/PROFILE only?

Reply:
Interesting. Yeah probably wouldn't hurt!